### ISSUE TITLE
Inline styles set in javascript

### ISSUE THREAD

**Issue by aronbos** [humhub/humhub#3982]
#3479 # Is your feature request related to a problem? Please describe.

Humhub does use a lot of inline styles.  That is very confusing if you make a theme. How does one know of a style is set in the stylesheet of by the program inline?
For example https://github.com/humhub/humhub/issues/3977 is probably caused by a missing max-height: 400px in the css file. Try searching and _I think_ the selection is scroll-able in that theme. (An inline max-height 400px is added.)

The real problem is in the order of CSS. It is according to https://stackoverflow.com/questions/6749569/css-which-takes-precedence-inline-or-the-class:

The order of precedence with CSS is as follows:
 >  **!important** (this is a bit hackish though but it is the only way to override an inline style. Try to avoid using this unless really necessary). Example: p {color: blue !important; }
>     **Inline**, such as p class="redText" style="color: red;" CSS is awesome /p.In this example, the class is ignored if the redText class declaration has already tried to define the property of color:. Other properties can still be honored though.
>     **Internal styles** - those written inside the <head><style> section of an html page.
>     **External stylesheet** which defines styles. Your html document must have a link to this sheet in order to use it. Example, again inside the <head> section is: <link rel="stylesheet" type="text/css" href="mystyle.css" />

So whatever I do in the theme, I can only override an inline style with !important. That is bad practice. Moreover, it is very difficult to find the cause, the offending inline style.

### Describe the solution you'd like
Do not set the style inline, but add or remove classes. 

So my proposal is to remove all inline styles step by step.

**Comment by luke-** [comment]
Yes, you're right .We should avoid inline styles wherever possible.

**Comment by buddh4** [comment]
We also need some more utility styles, some inline scripts depend on theme variables. Perhaps also use css variables in combination with https://stackoverflow.com/questions/46429937/ie11-does-a-polyfill-script-exist-for-css-variables

**Comment by aronbos** [comment]
I would only invest in the future and skip IE11. Everyone with IE11 can also use an other browser. Perhaps detect an IE11 browser and ask to use an modern browser with Humhub. I think, investing in IE11 is a waste a time.

**Comment by buddh4** [comment]
I'd love to get rid of IE support, but unfortunately some companies still stick with this browser in their software stack. At least this should be discussed first to find out who may still depends on IE support.
